subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,89 @@
+"""
+Summary
+-------
+Testing basic container as systemd service
+
+Operational Summary
+---------------------
+This is to test container as systemd service using a predefined unit file
+(``p4321.service``) and a ``script p4321-server.py``. An image is built with
+this script added. Systemd starts a container using this image & the container
+writes current time to port ``4321``. From host, the socket is read and the
+value is checked to see if it is a digit. Finally, container is stopped
+through systemd and system is cleaned up.
+
+"""
+
+import shutil
+import time
+import socket
+from os.path import exists
+from os import unlink
+from autotest.client import utils
+from autotest.client.shared.utils import is_port_free
+from dockertest.subtest import Subtest
+from dockertest.dockercmd import DockerCmd
+from dockertest.images import DockerImages
+from dockertest.output import mustpass
+from dockertest.containers import DockerContainers
+
+
+class systemd(Subtest):
+
+    def initialize(self):
+        super(systemd, self).initialize()
+        self.stuff['systemd_touchstone'] = False
+        if self.config['docker_path'] == '/usr/bin/docker':
```

Comment:
Aye, the only reason that config. option exists is to allow building docker (which requires docker) and then pointing at the built binary instead of the installed one.  Basically Ed's right, you can completely forget that the option even exists :grinning: